integrations/pgvector/src/haystack_integrations/document_stores/pgvector/document_store.py

```diff
@@ -91,7 +91,9 @@ def __init__(
         search_strategy: Literal["exact_nearest_neighbor", "hnsw"] = "exact_nearest_neighbor",
         hnsw_recreate_index_if_exists: bool = False,
         hnsw_index_creation_kwargs: Optional[Dict[str, int]] = None,
+        hnsw_index_name: str = HNSW_DEFAULT_INDEX_NAME,
```

Comment:
```suggestion
        hnsw_index_name: str = "haystack_hnsw_index",
```

Now that this parameter is configurable, I would specify the default value here to make things clearer and remove the constant.